editor/project_manager.cpp

```diff
@@ -904,36 +904,53 @@ class ProjectDialog : public ConfirmationDialog {
 			default_renderer_type = EditorSettings::get_singleton()->get_setting("project_manager/default_renderer");
 		}
 
+#if defined(GLES3_ENABLED) && (defined(WEB_ENABLED) || !defined(VULKAN_ENABLED))
```

Comment:
It's enough to check WEB_ENABLED IMO.

Reply:
The more complex the checks, the more chances we have to get them wrong now or in the future.